src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -264,29 +294,32 @@ void convertRequireToImportStatements(Node n, String fullLocalName, String requi
 
     if (module.providesObjectChildren.get(requiredNamespace).size() > 0) {
```

Comment:
It'll break the existing test cases; though, those cases may not actually be valid.

In any case, since it changes existing functionality/support, it should be in its own PR.